src/Console/Installer.php

```diff
@@ -105,6 +106,21 @@ public static function createDotEnv($dir, $io)
         }
     }
 
+    /**
+     * Copy README-dest.md into README.md
+     *
+     * @param string $dir The application's root directory.
+     * @param \Composer\IO\IOInterface $io IO interface to write to console.
+     * @return void
+     */
+    public static function createReadme($dir, $io)
+    {
+        $readme = sprintf('%s/README.md', $dir);
```

Comment:
Since `postInstall()` is executed every time `composer install` is launched the `README.md` will be always overridden at every install execution.

This operation should be done only at the first installation. We could remove `README-dest.md` after the first installation and check if file exists before copy it.

Reply:
ok. I guess that file should also be in .gitignore

Reply:
I was thinking that the right event to use for moving README-dest.md to README.md is post-create-project-cmd instead of post-install.
In this way we could remove that file from .gitignore and avoid the copy when we clone and install this project

Reply:
I'll apply a patch (maybe on monday) ;)